spaces/S000108/properties/P000139.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000108
+property: P000139
+value: true
+refs:
+  - zb: "0386.54001"
+    name: Counterexamples in Topology
+---
+
+The space {P139} because
+$\omega$ is a {P52} subspace.
+See e.g. {{zb:0386.54001}} space 111, item 1.
```

Comment:
```suggestion

See item #1 for space #111 in {{zb:0386.54001}}.
```
This is the usual way we phrase references to S&S (and in a separate paragraph).